c7n/query.py

```diff
@@ -540,6 +529,58 @@ def generate_arn(self):
         return self._generate_arn
 
 
+class MaxResourceLimit(object):
+
+    C7N_MAXRES_OP = os.environ.get("C7N_MAXRES_OP", 'or')
+    C7N_MAXRES_PERCENT = os.environ.get("C7N_MAXRES_PERCENT", 100)
```

Comment:
why is this an env var.. if you set it this breaks.. aka whats the type of environment variable.